content/blog/2018/10/2018-10-09-telemetry.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: >
+  Jenkins telemetry for higher quality changes
```

Comment:
Not really happy with the title.

Reply:
Improving Quality of Jenkins releases through real-world telemetry